client/elements/suttaplex/sc-suttaplex.html

```diff
@@ -317,14 +317,16 @@ <h3 class="suttaplex-heading" title="[[_getMainHeadingTitle(item, localize)]]">
               <a class="author-link" href="[[_getSuttaTextUrl(item.uid, translation.lang, translation.author_uid)]]">
                 <paper-button class="author-button" flat
                               title$="{{localize('goToTranslation', 'author', translation.author)}}">
-                    [[translation.author_short]]
+                  [[translation.author_short]]
                 </paper-button>
               </a>
             </template>
           </div>
-          <label class="translations-label">
-            {{localize('translations')}}
-          </label>
+          <template is="dom-if" if="[[_anyUserLanguageTranslations(item.translations, language)]]">
+            <label class="translations-label">
+              {{localize('translations')}}
+            </label>
+          </template>
```

Comment:
Thanks for fixing this one! I noticed it before and had meant to look into it. Great that's it's done!